draft-ietf-jsonpath-base.md

```diff
@@ -525,7 +525,7 @@ to the JSONPath processing (e.g., index values and steps) MUST be
 within the range of exact values defined in I-JSON {{-i-json}}, namely
 within the interval \[-(2<sup>53</sup>)+1, (2<sup>53</sup>)-1].
 
-2. Uses of function extensions must be *well typed*,
+2. Uses of function extensions must be *supported* and *well typed*,
```

Comment:
This section is about validity of a query.  Validity is independent of what you have implemented.  A query may be

* valid
* not valid
* of unknown validity (e.g., because you don't know a registration that might have occurred).

Reply:
Whether you actually support a valid query is a separate question.  (You can't really "support" invalid or unknown-valid-or-not queries.)

Reply:
So let me make this point clearer: This section is about queries and when they are valid.  It is now about implementations.

Reply:
Thanks for this important point. Another reason to abandon the PR.